### PR TITLE
Update gh-actions and CI pipeline to latest from keptn-service-template-go

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
Signed-off-by: Christian Kreuzberger <christian.kreuzberger@dynatrace.com>

This PR updates gh-actions, docker build, etc... to the latest and greatest from our keptn-service-template-go

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ FROM golang:1.13.7-alpine as builder
 
 RUN apk add --no-cache gcc libc-dev git
 
-WORKDIR /src/litmus-service
+WORKDIR /src/keptn-service-template-go
```
Comment (jetzlstorfer) [inline Dockerfile:8]: is this correct or should it be `src/litmus-service`?

Comment (jetzlstorfer) [reply Dockerfile:8]: same would be true for the rest of the file

**Review by jetzlstorfer — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply Dockerfile:8]: Thanks, will take a look

**Review by jetzlstorfer — APPROVED** [review]
lgtm - did not verify it locally but looked through it. looks good!

**Pull request merged**